src/scss/main.scss

```diff
@@ -259,3 +267,7 @@ textarea {
     }
   }
 }
+
+.visiblity-hidden {
+  visibility: hidden;
+}
```

Comment:
don't create a separate class for a single style

Reply:
why not use a global class instead of adding the same style in other classes?